src/components/icon/styles.css

```diff
@@ -1,3 +1,18 @@
+a.fill-div {
```

Comment:
We are using BEM Naming notation for all css classes — http://getbem.com/naming/

Reply:
`component__element`, where:
- component — has to be always equal to a folder name of this CSS file
- element — has to be a unique name within this file